### DISCUSSION TITLE
I have problems on using archinstall to install Arch

### DISCUSSION THREAD

**Discussion by FTS427** [archlinux/archinstall#2489] [Q&A]
![Image_1714655134058.png](https://github.com/archlinux/archinstall/assets/122330825/3d8d2863-8851-4e2c-b015-222229cd99a8)

Look this screenshot, I don't know what happened. I have inited pacman-key, but it still there.

Sorry for my English.

**Comment by svartkanin** [comment]
This looks like an older version of archinstall.
Xan you please use the latest version, if the problem persists then please share the log file as well/var/log/archinstall/install.log.
Also can you raise this as an issue not a discussion
